common/src/main/java/com/example/android/uamp/media/MusicService.kt

```diff
@@ -99,6 +109,12 @@ open class MusicService : MediaBrowserServiceCompat(), SessionAvailabilityListen
         BrowseTree(applicationContext, mediaSource)
     }
 
+    private val dataSourceFactory : DefaultDataSourceFactory by lazy {
+        DefaultDataSourceFactory(
+            /* context= */ this, Util.getUserAgent(/* context= */ this, UAMP_USER_AGENT), /* listener= */ null
```

Comment:
Since Kotlin has named arguments, I'd prefer it if you just wrote `context=this`, etc... rather than the comment format for Java :)

Reply:
This only works for Kotlin classes though :(

Reply:
Effectively, I cannot put it there.